client/landing/gutenboarding/onboarding-block/language/index.tsx

```diff
@@ -31,7 +31,7 @@ const LanguageStep: React.FunctionComponent< Props > = ( { previousStep } ) => {
 
 	const localizedLanguageNames = useSelect(
 		( select ) =>
-			( select( I18N_STORE ) as I18nSelect ).getLocalizedLanguageNames(
+			select( I18n.store ).getLocalizedLanguageNames(
```

Comment:
I'm surprised it wasn't used more! I guess this store could probably be removed after #74475